test/extension/rasterization/line.cpp

```diff
@@ -7,24 +6,24 @@
 // http://www.boost.org/LICENSE_1_0.txt)
 //
 
-#include <algorithm>
+#include <boost/gil/point.hpp>
+#include <boost/gil/extension/rasterization/line.hpp>
+
 #include <boost/core/lightweight_test.hpp>
-#include "boost/gil/point.hpp"
-#include "boost/gil/extension/rasterization/line.hpp"
 
+#include <algorithm>
 #include <cmath>
-#include <cstddef>
+#include <cstdint>
 #include <iterator>
 #include <random>
 #include <vector>
 
 namespace gil = boost::gil;
 
-namespace boost
-{
-namespace gil
-{
-std::ostream& operator<<(std::ostream& os, const point_t p)
+// TODO move into separate file to make it accessible also for other tests
+// operator<< overload is needed to use points directly with boost test macros
+namespace boost { namespace gil {
+std::ostream& operator<<(std::ostream& os, point_t const& p)
```

Comment:
You could follow our convention of `test_fixture.hpp`-s, create `test/core/point/test_fixture.hpp` and put this operator in there